pages/coinbase/dashboard.py

```diff
@@ -6,9 +6,19 @@
 
 class DashboardPage:
 
-    def __init__(self, driver, drivethrough=True):
+    def __init__(self, driver, spawn=False):
+        """
+        inputs
+        -----
+        driver = selenium driver
+        spawn: bool
+            The default use case is that a driver that is already at the 
+            appropriate url is passed in as part of a user workflow. if 
+            spawn=True, the driver will be navigated to the appropriate url on 
```

Comment:
Isn't more clear to say, navigated to dashboard on instantiation

Reply:
Since this is in every page object and has the same function in every page object, I want to keep it general.